docs/source/usage.rst

```diff
@@ -25,6 +25,9 @@ Input Arguments
       -start_snp START_SNP
                           The first marker to consider. The first marker is "1". Default: 1.
       -stop_snp STOP_SNP  The last marker to consider. Default: all markers considered.
+      -alt_allele_prob    The alternative allele frequencies per metafounder(s). Default: 0.5
```

Comment:
@RosCraddock @XingerTang I assume that we will provide a file here since it can be tedious to provide too many values in the console? Right? @RosCraddock modify the text accordingly.

Reply:
I assume it would be a file; I will correct the documentation. Would that change the -alt_allele_prob to this: 
`-alt_allele_prob_file [ALTERNATIVE ALLELE PROBABILITIES ...]` 
What are the [PEDIGREE...] etc for? @XingerTang @gregorgorjanc

Reply:
Yes, let say we need as many values as markers